examples/sol-anchor-contract/programs/example-sol-anchor-contract/src/state.rs

```diff
@@ -0,0 +1,41 @@
+use std::str::FromStr;
+use anchor_lang::prelude::*;
+use pyth_sdk_solana::state::load_price_account;
+
+use crate::ErrorCode;
+pub use pyth_sdk::Price;
+
+#[account]
+pub struct AdminConfig {
+    pub loan_price_feed_id:       Pubkey,
+    pub collateral_price_feed_id: Pubkey,
+}
+
+#[derive(Clone)]
+pub struct PythPrice {
+    pub price: Price,
```

Comment:
I preferred the price feed more 🤔 soon we can remove id from price feed and make it work. 

Also might be good to implement it as an unnamed wrapper and implement deref for it. (like anchor recommendation)

Reply:
+1

Reply:
I think it's better like this `pub struct PythPrice(Price);
impl Deref`

Reply:
But i like this better than Pricefeed so I disagree with Ali in that

Reply:
What if someone needs ema price? 🤔 another problem is that `get_price_{unchecked,no_older_than}` methods are defined on price feed level.

Reply:
I agree with Ali of using `PriceFeed` later after `PriceFeed` removes the key field, so that ema price is included.
I agree with Guillermo about using `pub struct PythPrice(Price)`. This is exactly how the Anchor originally recommended.

Lastly, Ali, I have found that `get_price_no_older_than` has been defined for both PriceAccount and PriceFeed. And their interface is slightly different. You may wish to unify the two.